jest/setup.js

```diff
@@ -119,6 +119,12 @@ const mockNativeModules = {
     addListener: jest.fn(),
     removeListeners: jest.fn(),
   },
+  Linking: {
+    openURL: jest.fn(),
+    canOpenURL: jest.fn(
+      (url) => new Promise((resolve) => resolve(url))
```

Comment:
This should return a boolean:

https://github.com/facebook/react-native/blob/master/Libraries/Linking/Linking.js#L164